ui/src/institutions/components/InstitutionAddressList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList, {
+  SEPARATOR_MIDDLEDOT,
+} from '../../common/components/InlineList';
+import InstitutionAddress from './InstitutionAddress';
+
+function renderAddress(address) {
+  return <InstitutionAddress address={address} />;
+}
+
+function InstitutionAddressList({ addresses }) {
+  return (
+    <InlineList
+      wrapperClassName="di"
+      items={addresses}
+      // FIXME: set extractKey explictly
```

Comment:
how about fixing this?

Reply:
I would but I'm not sure what to add to the key. Nothing in the address is unique/mandatory

Reply:
I see, it's not nice like that, It's calling `Map.toString()` all the time to get the key, we could memoize that but I guess it's not worth it at the moment.